apps/condo/domains/banking/access/BankTransaction.js

```diff
@@ -28,11 +24,12 @@ async function canReadBankTransactions ({ authentication: { item: user }, contex
         }
     }
 
+    const permittedOrganizations = await getEmployedOrRelatedOrganizationsByPermissions(user, [])
```

Comment:
What about function `getEmployedOrRelatedOrganizations`?

It seems that calling of `getEmployedOrRelatedOrganizationsByPermissions` without permissions is common

You have added similar functions `checkUserEmploymentInOrganizations`/`checkUserEmploymentOrRelationToOrganization`, why not add `getEmployeed...`/`getRelatedOrganizations` as well?